src/server/sams/api/consume/assets.py

```diff
@@ -50,6 +53,30 @@ def download_binary(asset_id):
     return response
 
 
+@assets_bp.route('/consume/assets/compressed_binary/<asset_ids>', methods=['GET'])
+def download_compressed_binary(asset_ids):
+    """
+    Uses asset_ids and returns the compressed
+    asset binaries zip
+    """
+    asset_ids = ast.literal_eval(asset_ids) if asset_ids else None
```

Comment:
Instead of using `literal_eval`, what if we were to be more strict in what we allow here? Such as specifically expecting a comma separated list of ids?